# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======

ReachGoalActionGoal action_goal
ReachGoalActionResult action_result
ReachGoalActionFeedback action_feedback
